### PR TITLE
Add periodic test for node-labels

### PR DESCRIPTION
**Pull request by ori-amizur**
Test for node-labels: new assisted-service feature

### CODE REVIEW

**Comment by adriengentil** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp
for sanity

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 1]
```diff
@@ -509,6 +509,24 @@ tests:
         NUMBER_OF_DAY2_HOSTS=1
       NUM_EXTRA_WORKERS: "2"
     workflow: assisted-baremetal-operator-ztp
+- as: e2e-ai-operator-ztp-node-labels-periodic
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:512]: shall we have a presubmit job as well ?

Comment (ori-amizur) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:512]: Added

**Comment by ori-amizur** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp
for sanity

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels

**Comment by ori-amizur** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-node-labels-periodic

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml [hunk 2]
```diff
@@ -1332,6 +1332,82 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )edge-e2e-ai-operator-ztp-multiarch-sno-ocp-411,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
[the remark this answers is not in the document]
Comment (ori-amizur) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml:1336]: This is a generated file.  Won't it be overwritten ?

**Review by adriengentil — APPROVED** [review]

**Pull request merged**